html/install.php

```diff
@@ -367,6 +391,9 @@
 //Please ensure this user is created and has the correct permissions to your install
 \$config['user'] = 'librenms';
 
+// This is the temporary directory used by php, set by php_admin_value[sys_temp_dir], defaults to /tmp when not set.
```

Comment:
Why are you are adding this to the default config.php? (which I plan to remove entirely at some point)
Setting it here actually doesn't allow php_admin_value[sys_temp_dir] to set it after you have installed LibreNMS

Reply:
In my test setup, I've set the following:
```
php_admin_value[open_basedir] = /var/www/myvhost
php_admin_value[sys_temp_dir] = /var/www/myvhost/tmp
```

With those settings, I still get php errors saying that /tmp isn't writable, unless I set
```
$config['temp_dir'] = /var/www/myvhost/tmp or sys_get_temp_dir();
```

So this is mainly a workaround for LibreNMS not honoring the tmp dir set by php_admin_value[sys_temp_dir].